bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,254 @@
+        :param space_after_s_delim:
+            If "\n", adds a line break after selector delimiter
+
+            Example:
+
+            ::
+                .a, .b{
+                    color: tomato;
+                }
+
+            After using the bear:
+
+            ::
+                .a,
+                .b{
+                    color: tomato;
+                }
+        :param space_before_s_delim:
+            If True, adds a space before selector delimiter.
+        :param space_before_c_brace:
+            If "\n", adds a line break before closing brace.
+        :param strip_spaces:
+            If True, trim trailing spaces.
+        :param tab_indent:
+            If True, use tab as indent.
+        :param vendor_prefix_align:
+            If True, align prefixes.
+
+            Example:
+
+            ::
+                .a {
+                    -webkit-border-radius: 3px;
+                    -moz- border-radius: 3px;
+                    border-radius: 3px;
+                }
+
+            After using the bear:
+
+            ::
+                .a {
+                    -hebkit-border-radius: 3px;
+                     -moz-border-radius: 3px;
+                       border-radius: 3px;
+                }
+        :param csscomb_json:
+            The path of the `.csscomb.json` config file.
+            If this option is present, all the above options
+            are not used. Instead, the `.csscomb.json` file is
+            used as the configuration file.
+        """
+
+        options = {"remove-empty-rulesets": remove_empty_rulesets,
+                   "always-semicolon": always_semicolon,
+                   "color-case": color_case,
+                   "block-indent": block_indent,
+                   "eof-newline": eof_newline,
+                   "leading-zero": leading_zero,
+                   "quotes": quotes,
+                   "sort-order-fallback": sort_order_fallback,
+                   "space-before-colon": space_before_colon,
+                   "space-after-colon": space_after_colon,
+                   "space-between-declarations": space_between_declaration,
+                   "space-before-opening-brace": space_before_o_brace,
+                   "space-after-opening-brace": space_after_o_brace,
+                   "space-after-selector-delimiter": space_after_s_delim,
+                   "space-before-closing-brace": space_before_c_brace,
+                   "strip-spaces": strip_spaces,
+                   "vendor-prefix-align": vendor_prefix_align
+                   }
+
+        if sort_order_fallback:
+            options["sort-order-fallback"] = "abc"
+        if space_before_o_brace:
+            options["space-before-opening-brace"] = "\n"
+        if space_after_o_brace:
+            options["space-after-opening-brace"] = "\n"
+        if space_before_colon:
+            options["space-before-colon"] = " "
+        if space_after_colon:
+            options["space-after-colon"] = " "
```

Comment:
I guess one of those branches isn't covered in your tests which is why coverage fails, check the codecov report